### PR TITLE
Alter the help message for test command to print the expected behaviour

### PR DESCRIPTION
**Pull request by misha-slyusarev**
This change adds additional information to the help message of `tofu test -test-directory` command (the help message is invoked by `tofu test --help`) to let the user know the command will search for test files both in the local folder and the one specified with the `-test-directory` flag.

Resolves https://github.com/opentofu/opentofu/issues/181

## Target Release

1.6.0

### ENHANCEMENTS

 

-  `tofu test --help`: description of the `-test-directory` flag includes expected behaviour

### CODE REVIEW

**Comment by misha-slyusarev** [comment]
@eranelbaz hey :) I see you opened another issue related to this code. I understand this should be a separate PR? 

I am unsure how to proceed with a review for this PR, though. Is anything still required from me, or should I wait for a maintainer to approve GitHub workflows here?

**Comment by eranelbaz** [comment]
Hey @misha-slyusarev 
Sorry for the closing, from my phone....
Currently we are focusing our efforts on the alpha release so this might wait for after it for the stable one.

IMO you can do `part of` for this help message as part of this pr

**Review by eranelbaz — APPROVED** [review]
Hey @misha-slyusarev going to merge this
If you would like to also grab https://github.com/opentofu/opentofu/issues/623 please communicate it on the issue 
Thank you

**Pull request merged**
